sdk/dotnet/Pulumi/Pulumi.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Does it mean that .NET Core 3.1 and .NET 5 users won't be able to use Pulumi anymore? Or will that still work?

Reply:
Yeah this is a breaking change. As discussed, i will be reverting this to maintain backward-compat